tests/components/recorder/test_history_db_schema_32.py

```diff
@@ -38,6 +38,17 @@ async def mock_recorder_before_hass(
     """Set up recorder."""
 
 
+@pytest.fixture
+def disable_states_meta_manager():
+    """Disable the states meta manager."""
+    with patch.object(
+        recorder.table_managers.states_meta.StatesMetaManager,
+        "active",
+        False,
+    ):
+        yield
```

Comment:
The `StatesMetaManager` is now always active in production, we set it to `False` to make the legacy history API queries run.